### ISSUE TITLE
Expo CLI versions 5.0.2+ not published to NPM

### ISSUE THREAD

**Issue by pweisensee** [expo/expo-cli#4089]
### Summary

Trying to upgrade to SDK 44, reading through the [upgrade steps in at the bottom of the blog post](https://blog.expo.dev/expo-sdk-44-4c4b8306584a) and it states:

> - Update to the latest version of Expo CLI: npm i -g expo-cli. expo-cli@5.0.2 or greater is required

However, the latest version published on NPM is version `5.0.1`. Is the version state incorrectly?

In the [Expo CLI change log it states `5.0.3` has been released](https://github.com/expo/expo-cli/blob/main/CHANGELOG.md#thu-16-dec-2021-154419--0800) and seems like we need it for SDK 44 compatibility

### Environment

```
  Expo CLI 5.0.1 environment info:
    System:
      OS: macOS 11.6.2
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 14.17.1 - ~/.nvm/versions/node/v14.17.1/bin/node
      Yarn: 1.22.10 - ~/.nvm/versions/node/v14.17.1/bin/yarn
      npm: 6.14.13 - ~/.nvm/versions/node/v14.17.1/bin/npm
      Watchman: 2021.12.20.00 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.11.2 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.2, iOS 15.2, macOS 12.1, tvOS 15.2, watchOS 8.3
    IDEs:
      Android Studio: 2020.3 AI-203.7717.56.2031.7935034
      Xcode: 13.2/13C90 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^43.0.0 => 43.0.4
      react: 17.0.1 => 17.0.1
      react-native: 0.64.3 => 0.64.3
    npmGlobalPackages:
      eas-cli: 0.42.4
      expo-cli: 5.0.1
    Expo Workflow: managed
```

### Please specify your device/emulator/simulator platform, model and version

All, running MacOS 11.6.2, NodeJS 14.17.1

### Error output

N/A

### Reproducible demo or steps to reproduce from a blank project

try to install latest version of Expo CLI

`npm i -g expo-cli`

Notice that it installs version `5.0.1` instead of `5.03`+

**Comment by darrylyoung** [comment]
I noticed this as well and asked in the Expo Discord but didn't get a response. Since then, I see that `5.0.4` has also been created but not published either.

@Kudo — is it recommended to install via `@next` and upgrade to SDK 44 that way or to wait for a published version that matches the `5.0.2` requirement mentioned in the blog post? Thanks!

**Comment by Kudo** [comment]
hi there! we've set `5.0.3` as `expo-cli@latest`.  thanks for being kind to report this issue.
